vendor/modules.txt

```diff
@@ -714,7 +667,6 @@ golang.org/x/crypto/ssh/agent
 golang.org/x/crypto/ssh/knownhosts
 golang.org/x/crypto/ssh/terminal
 # golang.org/x/net v0.0.0-20200226121028-0de0cce0169b
-## explicit
```

Comment:
Ugh, I think most of those changes are coming from go versions < 1.14. Can you please revert them?